overlay.d/05core/usr/lib/dracut/modules.d/35coreos-live/coreos-livepxe-rootfs.sh

```diff
@@ -26,7 +26,7 @@ elif [[ -n "${rootfs_url}" ]]; then
     if [[ ${rootfs_url} != http:* && ${rootfs_url} != https:* ]]; then
         # Don't commit to supporting protocols we might not want to expose in
         # the long term.
-        echo "coreos.live.rootfs_url= supports HTTP and HTTPS only." >&2
+        echo "coreos.live.rootfs_url=${rootfs_url} supports HTTP and HTTPS only." >&2
```

Comment:
Maybe here let's reword as e.g.

```suggestion
        echo "coreos.live.rootfs_url=${rootfs_url} uses unsupported scheme. Only HTTP and HTTPS are supported." >&2
```

?

Reply:
I like this. Adding it.

Reply:
I made it: `uses an unsupported scheme` WDYT?